src/templates/pages/example/form-validation/code.js

```diff
@@ -0,0 +1,65 @@
+import { h } from 'preact';
+
+export const FormValidation = () => <form class="form js-validate" action="#">

```

Comment:
Should the const be Code?
With the export default at the bottom, is this export required?

Reply:
Cheers - updated.